packages/web-react/src/components/Dropdown/demo/DropdownDefault.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Ref } from 'react';
+import { DropdownRenderProps } from '../../../types';
+import { Button } from '../../Button';
+import { Icon } from '../../Icon';
+import { Text } from '../../Text';
+import { Dropdown } from '..';
+
+const DropdownDefault = () => {
+  const dropdownTrigger = ({ trigger: { className, ref, ...restOf } }: DropdownRenderProps) => (
+    <Button UNSAFE_className={className} ref={ref as Ref<HTMLButtonElement>} {...restOf}>
+      Button as anchor
+    </Button>
+  );
+
+  const dropdownContent = (
+    <>
+      <a href="#info" className="d-flex mb-400">
+        <Icon name="info" />
+        &nbsp;<Text UNSAFE_className="ml-300">Information</Text>
+      </a>
+      <a href="#link" className="d-flex mb-400">
+        <Icon name="link" />
+        &nbsp;
+        <Text UNSAFE_className="ml-300">Bibendum aliquam, fusce integer sit amet congue non nulla aliquet enim</Text>
```

Comment:
This creates a <p> inside the <a>. Which shouldn't be.
Can we implement it the same way it is in twig?

```
<a href="#" class="d-flex mb-400">
            <Icon name="info" UNSAFE_className="mr-400" />
            <span>Information</span>
            ...
```